packages/teams-js/src/public/constants.ts

```diff
@@ -49,15 +61,23 @@ export enum HostName {
   teamsModern = 'TeamsModern',
 }
 
-// Ensure these declarations stay in sync with the framework.
+/** FrameContexts provides information about the context in which the app is running within the host. */
 export enum FrameContexts {
+  /** Allows developers to access and modify the configuration settings of a host. */
   settings = 'settings',
+  /** Provides access to the content of the current page or tab within the host. */
   content = 'content',
+  /** Provides methods for authenticating users and obtaining authentication tokens for host app. */
```

Comment:
>/** Provides methods for authenticating users and obtaining authentication tokens for host app. */ [](http://example.com/codeflow?start=2&length=98)

This doesn't seem accurate. This enum value doesn't provide methods for anything.